scenes/hadith.ts

```diff
@@ -36,17 +36,18 @@ scene.wait('hadith').on('message:text', async (ctx) => {
 })
 
 scene.wait('category').on(['message:text', 'callback_query:data'], async (ctx) => {
-  await ctx.answerCallbackQuery()
-
-  const category =
-    ctx?.message?.text == 'not' ? undefined : ctx?.message?.text ? ctx.message.text : ctx.callbackQuery?.data
-
   await Model.Hadith.create<IHadith>({
     content: ctx.session.hadith,
-    category,
+    category: ctx.callbackQuery?.data || ctx.update.message?.text,
   })
 
-  await ctx.editMessageText('The hadith is written, thank you. You are doing your best :)')
+  if (ctx.callbackQuery?.data) {
+    await ctx.answerCallbackQuery()
+
+    await ctx.editMessageText('The hadith is written, thank you. You are doing your best :)')
+  } else {
+    await ctx.reply('The hadith is written, thank you. You are doing your best :)')
```

Comment:
messageni constant qilib ishlatish kere

Reply:
Badge keremi halyam))

Reply:
tushunmadim?

Reply:
Just bind it to a variable and use it everywhere.

Reply:
Githubdigi badge midi ili bowqa narsami

Reply:
Manga suggestion berses bosvoraman dmoqciman agar kere bosa

Reply:
erinvosmi dimande 😂

Reply:
Yeah obedlanvomman